### ISSUE TITLE
Low accuracy

### ISSUE THREAD

**Issue by bsar321** [BindsNET/bindsnet#575]
Hello,
I'm trying to reproduce the diehl & cook network, using 6400 neurons in training, 15 epochs and 350ms but I don't get the accuracy you mentioned of 95% but 90% (excitatory weight=22.5 and inhibitory weight = 120), there is some parameter I need change?
I use the batch_eth_mnist.py file.
Thanks

**Comment by Hananel-Hazan** [comment]
Try to use the default parameters.

**Comment by bsar321** [comment]
Hi, 
With default parameters the accuracy obtained is 80% which is very low compared with the accuracy of paper. 
Can you help me ?

**Comment by Hananel-Hazan** [comment]
Which paper are you referring to?

**Comment by bsar321** [comment]
> 

I reffering to "BindsNET: A Machine Learning-Oriented Spiking Neural Networks Library in Python" , in section 4 (examples) more specific in subsection 4.1 is mentioned an accuracy of 95% but i can't get these accuracy.

**Comment by Hananel-Hazan** [comment]
Oh ok, please use `eth_mnist.py`. The script you use `batch_eth_mnist.py` is meant to show how to use the batch method and this script is not optimized for the best performance.

**Comment by bsar321** [comment]
Thank you, but before I used eth_mnist.py and the precision was 80% with the default parameters, what do I need to change to get the 95% ? thanks

**Comment by Hananel-Hazan** [comment]
The script used to reach 95% is `SOM_LM-SNNs.py` which is mentioned here `Unsupervised learning with self-organizing spiking neural networks`

**Comment by bsar321** [comment]
Ok, i will try, do you recommend using the default parameter or others to obtain 95%? Thanks again

**Comment by Hananel-Hazan** [comment]
I think the default parameters should get you higher accuracy then 80%. In the paper there is a table of the number of neurons and the their accuracy. I think the script should give you close enough to the paper. There are many changes in BindsNET since then, so your mileage may vary :-)
